### DISCUSSION TITLE
Difference between caption_dropout_rate vs caption_tag_dropout_rate?

### DISCUSSION THREAD

**Discussion by caniyabanci76** [kohya-ss/sd-scripts#1366] [Q&A]
What is the difference between caption_dropout_rate and caption_tag_dropout_rate?

**Comment by rockerBOO** [comment]
i believe `caption_dropout_rate` is the full caption (so rate where there is no caption) and `caption_tag_dropout_rate` is per each comma'd section `beautiful,amazing,wonderful` where it will remove these the tags per the dropout rate.
